developer/build_plugin.py

```diff
@@ -330,7 +331,7 @@ def _bake_manifest(manifest_data, config_uri, core_descriptor, plugin_root):
 
             fh.write("base_configuration=\"%s\"\n" % config_uri)
 
-            for (parameter, value) in manifest_data.iteritems():
+            for (parameter, value) in six.iteritems(manifest_data):
```

Comment:
I think that for most of our code case, calling `dict.items()` wouldn't be that big of a deal. Even tough it would be less efficient, we're not talking about thousands of elements being returned here, so it's not quite bad. It would make our lives easier in the far future when we remove six support because 2.7 is dead and burried for the industry and it would make presently our code easier on the eyes.

What do you think @manneohrstrom @willis102 ?

Reply:
I've gone ahead and made this change everywhere it seemed to make sense.  We might want to take a look and double check that there aren't any potential performance problems introduced, but I tried to keep an eye on that as I was making the changes.